CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### Enhancements
 
 - (react-native) Update bugsnag-android from v5.28.1 to [v5.28.3](https://github.com/bugsnag/bugsnag-android/blob/master/CHANGELOG.md#5283-2022-11-16)
+- (delivery-fetch) Create fetch based notifier delivery package [#1878](https://github.com/bugsnag/bugsnag-js/pull/1878)
+- (web-worker) Create web-worker notifier package to facilitate changes in google chrome extension development using [manifest v3](https://developer.chrome.com/docs/extensions/mv3/intro) [#1878](https://github.com/bugsnag/bugsnag-js/pull/1878)
```

Comment:
@gareththackeray happy to link to the chrome documentation here?

Reply:
IMO, I wouldn't mention browser extensions here at all. The feature is for web workers and has wider users beyond the browser extension part.